src/Components/ImagesTable/ImageBuildErrorDetails.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+
+const useGetErrorReason = (err) => {
+    if (!err || !err.reason) {
+        return 'An unknown error occured';
+    }
+
+    if (err.details && err.details.reason) {
+        return err.details.reason;
+    }
+
+    return err.reason;
+};
+
+const FailureIcon = () => (
```

Comment:
We could also use a PatternFly `<Alert>` here, specifically the inline plain variation:
https://www.patternfly.org/v4/components/alert/

Reply:
Ahhhh I didn't think of that. I'll update.